### ISSUE TITLE
react-github-calender breaks on firefox 

### ISSUE THREAD

**Issue by sudo-dliberty** [grubersjoe/react-github-calendar#42]
After adding to my project, testcases began to fail on firefox. 

On my particular project, the error I see is:

`SyntaxError: expected expression, got '.'`

Which directs to:

`  const year = getYear__default["default"](parseISO__default["default"](data[0]?.date));`

The example site provided also does not work on firefox:

https://grubersjoe.github.io/react-github-calendar/

<img width="1712" alt="Screen Shot 2022-12-02 at 7 53 11 PM" src="https://user-images.githubusercontent.com/57368453/205414294-00ec50f7-bd5f-41e2-8db5-eb8c88bc8be3.png">

**Comment by grubersjoe** [comment]
Hey. Unfortunately, I had a few issues with the upgrade to Rollup v3 and one broken version released. But it's fixed again in the latest release.

**Comment by sudo-dliberty** [comment]
Just checked, confirmed this is still not working on firefox 

Best to download the browser and check it out

**Comment by grubersjoe** [comment]
It did that after thefix. It works for me in Firefox:

![image](https://user-images.githubusercontent.com/3109364/206497893-cab618db-c347-4b21-8a2f-6a0c3d96e430.png)

Is there some caching issue maybe? I also tried a Firefox incognito window and it still workes fine.

**Comment by sudo-dliberty** [comment]
I had tried in incognito and cleared caches when I opened the ticket, so I knew this was okay.

Only other difference I noticed is that you are using FireFox on a windows machine, I am using it on my MacBook Pro. 

<img width="1710" alt="Screenshot 2022-12-08 at 11 47 27 AM" src="https://user-images.githubusercontent.com/57368453/206513279-16c6e54f-c954-4927-b1f5-d7e0102e258c.png">

So to test this out, I spun up a Windows 11 VM. Looks like it works on windows! 

<img width="1075" alt="Screenshot 2022-12-08 at 11 50 41 AM" src="https://user-images.githubusercontent.com/57368453/206513911-57c4a15a-0660-4b62-b350-f1123593ea55.png">

This really does not make sense to me though, OS should not matter. So then I looked at the FireFox versions. 

My MacBook had version 74.0.1, released April 3, 2020
The VM had version 107.0.1 released November 29, 2022.

Surprised I was that out of date, I updated the one on my Macbook to 107.0.1. Presto! started working.

TLDR, make sure browsers are up to date  👍

**Comment by grubersjoe** [comment]
Ah okay, that explains it of course ^^. Glad you figured it out.
